src/app/app.module.ts

```diff
@@ -1,5 +1,7 @@
-import { GA4GHService } from './shared/swagger/api/gA4GH.service';
-/*
+import { DescriptorLanguageService } from './shared/entry/descriptor-language.service';
+import { ErrorService } from './shared/error.service';
```

Comment:
FYI, there's some issue with your IDE placing imports before copyright notices

Reply:
Ya, I made an issue about it https://github.com/buehler/typescript-hero/issues/409.  I generally fix it when I notice it (there's been lots).  It'd probably be a 3 character fix for the plugin.  I wonder if I should make a PR to it...